src/apic-extension/azext_apic_extension/tests/latest/test_register_command.py

```diff
@@ -0,0 +1,115 @@
+            self.check('kind', 'rest'),
+            self.check('license.name', 'Apache 2.0'),
+            self.check('license.url', 'http://www.apache.org/licenses/LICENSE-2.0.html'),
+            self.check('lifecycleStage', 'design'),
+            self.check('name', 'swaggerpetstore-openapi30'),
+            self.check('summary', 'This is a sample Pet Store Server based on the OpenAPI 3.0 specification.  You can find out more about\nSwagger at [http://swagger.io](http://swagger.io). In the third iteration of the pet store, we\'ve'),
+            self.check('title', 'swagger-petstore---openapi-3.0'),
+        ])
+
+        self.cmd('az apic api version show -g {rg} -s {s} --api-id swaggerpetstore-openapi30 --version-id 1-0-19', checks=[
+            self.check('lifecycleStage', 'design'),
+            self.check('name', '1-0-19'),
+            self.check('title', '1-0-19'),
+        ])
+
+        self.cmd('az apic api definition show -g {rg} -s {s} --api-id swaggerpetstore-openapi30 --version-id 1-0-19 --definition-id openapi', checks=[
+            self.check('description', 'This is a sample Pet Store Server based on the OpenAPI 3.0 specification.  You can find out more about\nSwagger at [http://swagger.io](http://swagger.io). In the third iteration of the pet store, we\'ve switched to the design first approach!\nYou can now help us improve the API whether it\'s by making changes to the definition itself or to the code.\nThat way, with time, we can improve the API in general, and expose some of the new features in OAS3.\n\nSome useful links:\n- [The Pet Store repository](https://github.com/swagger-api/swagger-petstore)\n- [The source API definition for the Pet Store](https://github.com/swagger-api/swagger-petstore/blob/master/src/main/resources/openapi.yaml)'),
+            self.check('name', 'openapi'),
+            self.check('specification.name', 'openapi'),
+            self.check('specification.version', '3-0-2'),
+            self.check('title', 'openapi'),
+        ])
+
+        self.cmd('az apic api definition export-specification -g {rg} -s {s} --api-id swaggerpetstore-openapi30 --version-id 1-0-19 --definition-id openapi --file-name test_register_with_json_spec.yml')
+
+        try:
+            exported_file_path = "test_register_with_json_spec.yml"
+            with open(exported_file_path, 'r') as file:
```

Comment:
if file not exist, is exception?

Reply:
Yes, the test case will fail with file not exist error. So I didn't check whether the file exist or not.